WcaOnRails/config/locales/en.yml

```diff
@@ -1265,17 +1265,17 @@ en:
       name_html: "WCA Board"
       info: "(WCA organization, delegate reports, competition approval, general inquiries)"
```

Comment:
Since we changed all the other `info`s to be complete sentences, do you think we should change this one as well?